### DISCUSSION TITLE
unsupported linux/amd64 ?

### DISCUSSION THREAD

**Discussion by vipul-singh-dezerv** [apache/superset#26777] [Q&A / Help]
i was redeploying superset today, deployment fails with following error :

│   Warning  Failed     16s (x2 over 30s)  kubelet            Failed to pull image "apache/superset:dockerize": rpc error: code = Unknown desc = no matching manifest for linux/amd64 in the manifest list entries

Shouldn't    'linux/amd64' be a supported architecture ?

**Comment by sprudel79** [comment]
I am facing the same issue when I try to run a local K8s cluster via kind and install the helm chart from [http://apache.github.io/superset/](http://apache.github.io/superset/) repos.
